test/integration/command_test.exs

```diff
@@ -110,7 +110,22 @@ defmodule Integration.CommandTest do
   end
 
   test "running a command in a pipeline with nil output", %{user: user} do
-    response = send_message(user, ~s(@bot: seed '[{"a": "1", "b": "2"}, {"a": "3"}]' | filter --field="b" | echo $a))
+    response = send_message(user, ~s(@bot: seed '[{"a": "1", "b": "2"}, {"a": "3"}]' | filter --path=b | echo $a))
     assert response["data"]["response"] == "1"
   end
+
+  test "reading the path to filter a certain path", %{user: user} do
+    response = send_message(user, ~s(@bot: seed '[{"foo":{"bar":{"baz":"stuff"}}}, {"foo": {"bar":{"baz":"me"}}}]' | operable:filter --path="foo.bar"))
+    assert response["data"]["response"] == "{\n  \"bar\": {\n    \"baz\": \"stuff\"\n  }\n}\n{\n  \"bar\": {\n    \"baz\": \"me\"\n  }\n}"
```

Comment:
Yes, the input / output pairs you specified just now are correct. 

Conceptually the `filter` command is just deciding whether or not to allow a given input map through to the rest of the pipeline; it should not transform any individual input map in any way (and this is why the `--return` option is being removed in favor of a `pluck` command in #255)